WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsInsightsDetailsViewModel.swift

```diff
@@ -634,7 +636,7 @@ private extension SiteStatsInsightsDetailsViewModel {
     func queryForInsightStatSection(_ statSection: StatSection) -> InsightQuery? {
         switch statSection {
         case .insightsFollowersWordPress, .insightsFollowersEmail, .insightsFollowerTotals:
-            return .allFollowers
+            return .insights // use .insights here which is same as top level insights screen
```

Comment:
Is a change like this replicable for other types of sections (like comments)? Or are they specific and this change is only relevant to followers?

I think this question boils down to the difference between `getAllDotComFollowers` vs `getDotComFollowers`.

Reply:
.allFollowers returns `getAllDotComFollowers` and for insights, we don't need to make a large-ish network call for AllDotComFollowers we just need the top 10 per screen designs which is provided by the fetchInsights call ie Insights is more of a summary.

Reply:
Okay got it now. Thanks for the explanation 👍